.travis.sh

```diff
@@ -12,7 +12,12 @@ set -e # Error build immediately if install script exits with non-zero
 # if this is a docker job, run in the container instead; but if not just run it here.
 if [ -n "$DOCKER_IMAGE" ]; then
     ./util/dockerize.sh ./.ci.sh
+elif [ "${TRAVIS_OS_NAME}" = "windows" ]; then
+    choco install wsl-ubuntu-1804
```

Comment:
This seems to install Ubuntu in a way that can only be accessed by Administrators. Presumably none of our users will be using WSL this way. This subtle distinction might bite us.

Reply:
There's an answer down below on how to do it, I suspect it's not critical, but if you'd like to investigate, go for it.